### ISSUE TITLE
Grid - WAVE shows missing accessibility attributes

### ISSUE THREAD

**Issue by AlisherAmonulloev** [DevExpress/devextreme-reactive#2417]
- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### I'm using ...

- [x] React Grid
- [ ] React Chart
- [ ] React Scheduler

### Current Behaviour

How to add Labels for Checkbox. Below is sample code which we are using
```js
<TableSelection
selectByRowClick
highlightRow
showSelectAll />

```
How to avoid empty table header. Below is sample code which we are using
```js
<TableHeaderRow
showSortingControls />
```

### Expected Behaviour

Table header should not be empty. Checkboxes should have labels.

### Screenshots

<img width="956" alt="Capture1" src="https://user-images.githubusercontent.com/25052481/66490545-ff9f7380-eab9-11e9-92f0-ca593f7fddd2.png">

P.S. This issue is created from the https://supportcenter.devexpress.com/ticket/details/T750617 ticket

**Comment by LazyLahtak** [comment]
Hi

To add labels for checkboxes, you can create your own [TableSelection.Cell](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-grid-material-ui/src/templates/table-select-cell.jsx) and [TableSelection.HeaderCell](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-grid-material-ui/src/templates/table-select-all-cell.jsx) components based on our code. Then, provide custom components to TableSelection's cellComponent and headerCellComponent properties.
To avoid an empty table header in filter cells, create your own [TableFilterRow.Cell](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-grid-material-ui/src/templates/table-filter-cell.jsx) component and provide it to TableFilterRow's cellComponent.
The component should contain some element provided as a cell's child. To avoid the empty table header in the stub cell, you need to do same with the Table.HeaderStubCell component and provide it to the Table`s stubHeaderCell property.
We prepared a [demo](https://codesandbox.io/s/devextreme-react-grid-for-material-ui-6pjto) illustrating how to do that. Let us know if it meets your needs.
